content/about/releases.md

```diff
@@ -0,0 +1,18 @@
+---
+authors: ["kokkos-team"]
+title: "Releases"
+date: "2024-08-11"
+tags: ["Release"]
+---
+
+# Kokkos Ecosystem release page
+
+## GnuPG signing keys
+
+The maintainers of the libraries sign the tags associated with releases and sign the source packages. In order to verify the authenticity of the source packages you can download the public keys here: [dalg24.asc](content/siging-keys/release-keys.asc) and [lucbv.asc](https://kokkos.org/downloads/signing-keys/lucbv.asc).
```

Comment:
The source packages and git tags are signed by the release managers. You can download the keys in order to verify [here](keys here).